.github/tailor.yaml

```diff
@@ -0,0 +1,10 @@
+rules:
+  - name: commit title
+    description: all commit titles must have a scope, be lowercase, and be no more than 50 characters
+    expression:  |-
+      .commits all(((.message test "^[a-z/\\-\\*]+: [a-z0-9 \\-'./]+") and (.message test "^.{0,50}\\n")) or (.message test "^Revert"))
```

Comment:
Re uppercase letters, I think it's useful to be able to correctly spell Go type/function/variable names in commit summaries.

Reply:
Makes sense. I chose that scheme based on how I personally write them, but I can certainly change it.